plugins/data-management/src/HierarchicalTrackSelectorWidget/model.ts

```diff
@@ -49,6 +63,31 @@ export default function stateTreeFactory(pluginManager: PluginManager) {
       view: types.safeReference(
         pluginManager.pluggableMstType('view', 'stateModel'),
       ),
+      /**
+       * #property
+       */
+      favorites: types.optional(
+        types.array(types.string),
+        () =>
+          JSON.parse(localStorageGetItem(localStorageKeyF()) || '[]').favorites,
+      ),
+      /**
+       * #property
+       */
+      recentlyUsed: types.optional(
+        types.array(types.string),
+        () =>
+          JSON.parse(localStorageGetItem(localStorageKeyF()) || '[]')
+            .recentlyUsed,
+      ),
+      /**
+       * #property
+       */
+      showRecentlyUsedCategory: true,
```

Comment:
looks like these settings would be shared via share links. it might be worth converting to a localstorage preference

Reply:
currently have them separate localstorage items, let me know if you want me to combine them into a single one